ider/src/ider.component.spec.ts

```diff
@@ -25,11 +25,6 @@ describe('IderComponent', () => {
 
   it('should create', () => {
     expect(component).toBeTruthy()
-    expect(component.ider).toBeInstanceOf(AMTIDER)
```

Comment:
shouldnt these expectations move to another test ?

Reply:
done